pkg/actors/actors.go

```diff
@@ -67,10 +67,11 @@ type actorsRuntime struct {
 }
 
 const (
-	idHeader        = "id"
-	lockOperation   = "lock"
-	unlockOperation = "unlock"
-	updateOperation = "update"
+	idHeader               = "id"
+	lockOperation          = "lock"
+	unlockOperation        = "unlock"
+	updateOperation        = "update"
+	incompatibleStateStore = "state store does not support transactions which actors require to save state - please see https://github.com/dapr/docs/blob/master/concepts/actor/actors_features.md#actor-state-management"
```

Comment:
I don't know about putting the actual link here.. there's already talks about restructuring some of our docs. I'd remove it, and have it be something like `check https://github.com/dapr/docs for more info`.

Reply:
This has the other problem that it may take the user a while to find the appropriate doc, but agree with the issue if links change.